splink/linker.py

```diff
@@ -2148,6 +2136,59 @@ def _compute_cluster_metrics(
 
         return df_cluster_metrics
 
+    def _compute_metrics_clusters(
+        self,
+        df_node_metrics: SplinkDataFrame,
+    ) -> SplinkDataFrame:
+
+        sqls = _size_density_centralisation_sql(
+            df_node_metrics,
+        )
+
+        for sql in sqls:
+            self._enqueue_sql(sql["sql"], sql["output_table_name"])
+
+        df_cluster_metrics = self._execute_sql_pipeline()
+        return df_cluster_metrics
+
+    # a user-facing function, which is currently 'private' (Beta functionality)
+    # while functionality is developed, as breaking changes may occur
+    def _compute_cluster_metrics(
+        self,
+        df_predict: SplinkDataFrame,
+        df_clustered: SplinkDataFrame,
+        threshold_match_probability: float,
+    ) -> Dict[str, SplinkDataFrame]:
+        """
+        Generates a table containing cluster metrics and returns a Splink dataframe
+
+        Args:
+            df_predict (SplinkDataFrame): The results of `linker.predict()`
+            df_clustered (SplinkDataFrame): The outputs of
+                `linker.cluster_pairwise_predictions_at_threshold()`
+            threshold_match_probability (float): Filter the pairwise match predictions
+                to include only pairwise comparisons with a match_probability above this
+                threshold.
+
+        Returns:
+            dict[str, SplinkDataFrame]: A dictionary of SplinkDataFrames
+                containing cluster IDs and selected cluster, node, or edge metrics
+                key "nodes" for nodes metrics table
+                key "edges" for edge metrics table
+                key "clusters" for cluster metrics table
+
+        """
+        df_node_metrics = self._compute_metrics_nodes(
+            df_predict, df_clustered, threshold_match_probability
+        )
+        # don't need edges as information is baked into node metrics
+        df_cluster_metrics = self._compute_metrics_clusters(df_node_metrics)
+
+        return {
+            "clusters": df_cluster_metrics,
+            "nodes": df_node_metrics,
+        }
```

Comment:
Minor suggestion to flip the ordering so that it matches the order of function calls above and table creation